src/plugins/controls/connectivity/pauseOnError.js

```diff
@@ -63,6 +63,9 @@ export default pluginFactory({
             }
         };
 
-        testRunner.before('error', (e, error) => processError(error));
+        testRunner.before('error', (e, error) => {
+            testRunner.trigger(`disablefeedbackalerts`);
```

Comment:
**nitpick:** Caution, this may prevent legitimate error notification. Please consider scoping it more.

**suggestion:** This should be done only when needed, and reverted as soon as possible.
As a suggestion, you could do something like this:
- when error occurs, if this is a 500, then prevent feedback alerts
- then, register an action to revert the feedback preventing once the error was processed
This could be done line 60, like so:
```javascript
            if (error.code === 500) {
                error.originalCode = error.code;
                delete error.code;

                testRunner.trigger(`disablefeedbackalerts`);
                testRunner.after('error.pauseOnError', () => {
                    testRunner.off('error.pauseOnError');
                    testRunner.trigger(`enablefeedbackalerts`);
                });
            }
```

Reply:
nice catch, thank you